app/grants/views.py

```diff
@@ -842,3 +842,22 @@ def is_verified(tx_details, tx_hash, tx_amount, network):
         )
 
     return get_json_response("Wrong request.", 400)
+
+
+def invoice(request, contribution_pk):
+    contribution = Contribution.objects.prefetch_related('subscription', 'subscription__grant').get(pk=contribution_pk)
+
+    # only allow invoice viewing if admin or if grant contributor
+    is_contributor = request.user.username.lower().lstrip('@') == contribution.subscription.contributor_profile.handle.lower().lstrip('@')
```

Comment:
hmm.. probably can do this logic simpler like `has_view_privs = request.user.is_staff or request.user.profile == contribution.subscription.contributor_profile`

Reply:
Cool,  i just changed!